README.md

```diff
@@ -1,39 +1,39 @@
-<!DOCTYPE html>
-<html>
-<body>
   <!-- HEADER/NAME -->
   <h1 id="name"><span style="font-variant: small-caps;">Ewon Ignition Demo</span></h1>
-
   <!-- TABLE OF CONTENTS -->
   <h2 id="toc"><span style="font-variant: small-caps;">Table of Contents</span></h2>
-
   <ol>
-    <li><a href="#information">Information</a>
+    <li>
+      <a href="#information">Information</a>
       <ol>
         <li><a href="#about">About</a></li>
         <li><a href="#system-design">System Design</a></li>
         <li><a href="#trademark-information">Trademark Information</a></li>
       </ol>
     </li>
-    <li><a href="#getting-started">Getting Started</a>
+    <li>
+      <a href="#getting-started">Getting Started</a>
       <ol>
         <li><a href="#prerequisites">Prerequisites</a></li>
       </ol>
     </li>
-    <li><a href="#installation">Installation</a>
+    <li>
+      <a href="#installation">Installation</a>
       <ol>
         <li><a href="#ewon-setup">Ewon Setup</a></li>
         <li><a href="#ignition-setup">Ignition Setup</a></li>
       </ol>
     </li>
     <li><a href="#usage">Usage</a></li>
-    <li><a href="#troubleshooting">Troubleshooting</a>
+    <li>
+      <a href="#troubleshooting">Troubleshooting</a>
       <ol>
         <li><a href="#cannot-log-in-trial-expired">Cannot log in, trial expired</a></li>
-        <li><a href="#error-writing-to-tag-bad_notfound">Error writing to tag, Bad_NotFound<a/></li>
+        <li><a href="#error-writing-to-tag-bad_notfound">Error writing to tag, Bad_NotFound</a></li>
       </ol>
```

Comment:
this fix is kinda out of scope for your commit message of: Add the necessary Jekyll configuration file and gem plugins to support GitHub Pages for README.md and fix README.md to remove conflicting DOCTYPE tag. instead of adding to the list you could just add it to a separate commit